sources/common/test/unitTest/FSignal.cpp

```diff
@@ -39,10 +39,12 @@ static inline std::string getFSignalPrint(const std::string &value)
 /// Other parameters are dummies.
 static inline SIGNAL::FSignal getFSignal(const std::string &value)
 {
-    return {DummySignal::DUMMY_NAME,
+    static std::vector<std::unique_ptr<const SIGNAL::SignalDescriptor>> vDesc = {};
+    vDesc.push_back(std::make_unique<const SIGNAL::SignalDescriptor>(DummySignal::DUMMY_NAME, DummySignal::DUMMY_SIZE, DummySignal::DUMMY_TYPE, DummySignal::DUMMY_HANDLE));
+
+    return { vDesc.back().get(),
```

Comment:
Same as above.

Reply:
Just fixed it here. No needed for FSignal.